### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by YoriB**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<section>
+<h2 id="underlined">COMMENTS SECTION</h2>
+
+<ul className="">
+      {comments.map((comment) => {
+return (
+  <li className = "comment-box" key={comment.comment_id}>
+<p>{comment.author}</p>
+<p>{comment.body}</p>
+<p>{comment.created_at}</p>
```
Comment (itskatherine) [inline src/components/Comments.jsx]: This date format isn't super readable at the moment, you could reformat this yourself, or have a library like Day.js do the hard work

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { getCommentsByArticleIdApi } from '../utils/api';
+
+const Comments =({singleArticleID}) => {
+  
+  const [comments, setComments] = useState([]);
+  
+
+  useEffect(() => {
+    if (singleArticleID) {
+    getCommentsByArticleIdApi(singleArticleID).then((CommentsFromArticleById) => {
```
Comment (itskatherine) [inline src/components/Comments.jsx]: Avoid naming variables with capital letters please, as that tends to indicate classes, or components 🙂

File: src/utils/api.js [hunk 3]
```diff
@@ -18,4 +18,9 @@ return data;
   });
 };
 
+export const getCommentsByArticleIdApi = (article_id) => {
+  return articlesAPI.get(`/articles/${article_id}/comments`).then(({ data }) => {   
+   return data;
+  })
+}
```
Comment (itskatherine) [inline src/utils/api.js:33]: Looks good 👍

**Review by itskatherine — COMMENTED** [review]
Looks good Yori, have a look over my comments and then feel free to merge 👍

**Review by itskatherine — APPROVED** [review]
Looks good Yori, have a look over my comments and then feel free to merge 👍

File: src/components/Comments.jsx (hunk 1 above)
Comment (YoriB) [reply src/components/Comments.jsx]: Ok will look at Day.js to sort it out.

File: src/components/Comments.jsx (hunk 2 above)
Comment (YoriB) [reply src/components/Comments.jsx]: Ok will do so going forwards..

**Pull request merged**